src/DependencyInjection/DI.External.Tests/test/Microsoft.Extensions.DependencyInjection.ExternalContainers.Tests.csproj

```diff
@@ -21,6 +22,8 @@
     <Reference Include="Grace.DependencyInjection.Extensions" />
     <Reference Include="Stashbox.Extensions.Dependencyinjection" />
     <Reference Include="Unity.Microsoft.DependencyInjection" />
+    
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
Why can't this project use a `@(Reference)` item like others in this PR❔

Reply:
This project uses the 2.1 era package brought in by `Unity.Microsoft.DependencyInjection` & others, upgrading it to one in the same band seemed like the minimal change